### PR TITLE
[ConvAI] Add information about override settings

### PR DESCRIPTION
**Pull request by aarthificial**

### CODE REVIEW

File: conversational-ai/customization/client-tools.mdx [hunk 1]
```diff
@@ -37,7 +37,7 @@ Follow these steps to set up a client-side event that triggers a browser alert.
     Depending on your development environment, add the following code to integrate the client tool.
 
     <CodeGroup>
-      ```javascript
+      ```javascript JavaScript
```
Comment (hikmet-demir) [inline conversational-ai/customization/client-tools.mdx:40]: good catch

File: conversational-ai/customization/conversation-configuration.mdx [hunk 2]
```diff
@@ -28,7 +28,13 @@ The following steps will show you how to configure the `prompt` & `first_message
         Ensure you have the latest [Javascript](https://elevenlabs.io/docs/conversational-ai/libraries/conversational-ai-sdk-js)/[React SDK](https://elevenlabs.io/docs/conversational-ai/libraries/conversational-ai-sdk-react) installed.
     </Tip>
     <Steps>
-    <Step>
+    <Step title="Enable overrides">
```
Comment (hikmet-demir) [inline conversational-ai/customization/conversation-configuration.mdx:31]: this part is same for no-matter the language, we should put it before language selection I would say

**Review by hikmet-demir — APPROVED** [review]
Thanks for the update

File: conversational-ai/customization/conversation-configuration.mdx (hunk 2 above)
Comment (aarthificial) [reply conversational-ai/customization/conversation-configuration.mdx]: Changed!

**Pull request merged**